shellpython/shellpy.py

```diff
@@ -3,20 +3,21 @@
 import os
 import subprocess
 from shellpython.preprocessor import preprocess_file
+from argparse import ArgumentParser
+import re
 
 
 def main():
-    if len(sys.argv) == 1 or sys.argv[1] in ['-h', '--help', 'help']:
-        print('Shellpy, write shell scripts in Python easily')
-        print('Usage: shellpy [*.spy script] [arguments]')
-        exit(0)
+    def shellpyFile(fil):
```

Comment:
arguments could be added with add_argument() function. what's the problem with that?